src/messages/javascript.js

```diff
@@ -124,26 +124,12 @@ export const UNSUPPORTED_API = {
   description: i18n._('This API has not been implemented by Firefox.'),
 };
 
-function deprecatedAPI(api) {
-  return {
-    code: apiToMessage(api),
-    message: i18n._(`"${api}" is deprecated or unimplemented`),
-    description: i18n._(oneLine`This API has been deprecated by Chrome
-      and has not been implemented by Firefox.`),
```

Comment:
Could we use a mechanism similar to the one we are now using in this PR to customize the deprecated warning message for the LWT aliases to also customize the message for the API that used to be pointed as unimplemented? 
(I mean only the following ones: 'app.getDetails', 'extension.onRequest', 'extension.onRequestExternal', 'extension.sendRequest', 'tabs.getAllInWindow', 'tabs.getSelected', 'tabs.onActiveChanged', 'tabs.onSelectionChanged', 'tabs.sendRequest').

This deprecated APIs are bit different from deprecations like the proxy API, because they were already deprecated in chrome when we started to work on the WebExtensions API and so they have never been implemented, personally I would prefer their warning message to possibly still state that. 

(I don't see it as a blocker, but it would be nice and at a first glance it seems to me that this PR shouldn't be changed too much to achieve that)